### PR TITLE
EZP-31021: Revised exception and command messages

### PR DESCRIPTION
**Pull request by DominikaK**
[EZP-31021](https://jira.ez.no/browse/EZP-31021)
Main PR: https://github.com/ezsystems/ezpublish-kernel/pull/2878

### CODE REVIEW

File: bundle/Command/SolrCreateIndexCommand.php [hunk 1]
```diff
@@ -60,15 +60,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $purge = !$input->getOption('no-purge');
         $bulkCount = $input->getArgument('bulk_count');
         if (!is_numeric($bulkCount) || (int)$bulkCount < 1) {
-            throw new RuntimeException("'bulk_count' argument should be > 0, got '{$bulkCount}'");
+            throw new RuntimeException("The 'bulk_count' argument should be > 0, you provided '{$bulkCount}'");
```
Comment (michalmilc90) [inline bundle/Command/SolrCreateIndexCommand.php]:
```suggestion
            throw new RuntimeException("The 'bulk_count' argument should be > 0. You provided '{$bulkCount}'");
```

**Comment by adamwojs** [comment]
@DominikaK Unit tests need to be updated here.

**Comment by DominikaK** [comment]
@adamwojs unit tests pass

**Comment by DominikaK** [comment]
Passing build with linked ezpublish-kernel changes (https://github.com/ezsystems/ezpublish-kernel/pull/2878): https://travis-ci.org/ezsystems/ezplatform-solr-search-engine/builds/615020744

**Pull request merged**